server/mcp_server_las/requirements.txt

```diff
@@ -108,7 +108,7 @@ uvicorn==0.34.0
     # via mcp
 volcengine==1.0.180
     # via mcp-server-las (pyproject.toml)
-mcp==1.9.4
+mcp==1.12.0
```

Comment:
wrong place to update version of dependency,  you have to update 
https://github.com/volcengine/mcp-server/blob/main/server/mcp_server_las/pyproject.toml indeed.

Reply:
generate requirement.txt by pyproject.toml